### DISCUSSION TITLE
Getting Prediction  Error: AttributeError: 'StatsForecast' object has no attribute 'fitted_'

### DISCUSSION THREAD

**Discussion by guzinerdem** [Nixtla/statsforecast#743] [Q&A]
Hi, 
After training and saving models with statsforecast, when I try to make prediction with sf.predict()  I get  AttributeError: 'StatsForecast' object has no attribute 'fitted_', how can I make prediction easily?

**Comment by jmoralez** [comment]
Hey @guzinerdem, thanks for using statsforecast. Can you please provide the code that you're running?

**Comment by Lineker98** [comment]
I am getting the same error, following the instructions to use Mlflow, it is not pissible to fit the model, when a call sf.fit(), i get 'StatsForecast' object has no attribute "'ga'". And when i log the model without fit, and trying to predict, i get 'StatsForecast' object has no attribute 'fitted_'

AttributeError: 'StatsForecast' object has no attribute 'fitted_'
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
File <command-67063469771845>:1
----> 1 results = loaded_model.predict(h=HORIZON, X_df=X_test, level=LEVEL)

File /local_disk0/.ephemeral_nfs/cluster_libraries/python/lib/python3.9/site-packages/statsforecast/core.py:976, in _StatsForecast.predict(self, h, X_df, level)
    974 X, level = self._parse_X_level(h=h, X=X_df, level=level)
    975 if self.n_jobs == 1:
--> 976     fcsts, cols = self.ga.predict(fm=self.fitted_, h=h, X=X, level=level)
    977 else:
    978     fcsts, cols = self._predict_parallel(h=h, X=X, level=level)

AttributeError: 'StatsForecast' object has no attribute 'fitted_'

**Reply by jmoralez** [reply]
Are you following [this guide](https://nixtlaverse.nixtla.io/statsforecast/docs/tutorials/mlflow.html)? I'm able to run it end to end (except I just need to change the import from `import mlflavors` to `import mlflavors.statsforecast`)
